### ISSUE TITLE
"in" capitalization after titles with punctuation

### ISSUE THREAD

**Issue by jackweinbender** [dcpurton/biblatex-sbl#102]
The situation can be easily observed in the footnote below. I believe the highlighted "In" should be lowercase, and I'm guessing that the cause of the problem is that the article title ends with a question mark:

![image](https://user-images.githubusercontent.com/1544859/60361290-75684f00-99a3-11e9-98b9-ab8d01608758.png)

    @incollection{carretero-alphen_wagoner2017,
      author     = {Carretero, Mario and van Alphen, Floor},
      crossref   = {wagoner2017},
      pages      = {283--303},
      shorttitle = {History},
      title      = {History, Collective Memories, or National Memories?},
    }
    @incollection{wertsch_wagoner2017,
      author     = {Wertsch, James V.},
      crossref   = {wagoner2017},
      pages      = {259--281},
      shorttitle = {National Memory},
      title      = {National Memory and Where to Find It},
    }
    @collection{wagoner2017,
      options    = {skipbib=1},
      editor     = {Wagoner, Brady},
      booktitle  = {Handbook of Culture and Memory},
      date       = {2017},
      shorttitle = {Handbook of Culture and Memory},
      title      = {Handbook of Culture and Memory},
      xdata      = {pub_oxford},
    }

Thanks as always.

**Comment by dcpurton** [comment]
Duplicate of issue #81.

**Comment by dcpurton** [comment]
Confirmed fixed in https://github.com/dcpurton/biblatex-sbl/commit/7bbe6361645d0daa692e2242d9e5032f2ccb86f2.

Current output is:

```TeX
\documentclass{article}
\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@misc{pub_chicago,
  location   = {Chicago},
  publisher  = {Chicago University Press}
}
@misc{pub_oxford,
  location   = {Oxford},
  publisher  = {Oxford University Press}
}
@book{redfield1956,
  author     = {Redfield, Robert},
  date       = {1956},
  title      = {Peasant Society and Culture},
  subtitle   = {An Anthropological Approach to Civilization},
  xdata      = {pub_chicago}
}
@incollection{carretero-alphen_wagoner2017,
  author     = {Carretero, Mario and van Alphen, Floor},
  crossref   = {wagoner2017},
  pages      = {283-303},
  shorttitle = {History},
  title      = {History, Collective Memories, or National Memories?},
}
@incollection{wertsch_wagoner2017,
  author     = {Wertsch, James V.},
  crossref   = {wagoner2017},
  pages      = {259-281},
  shorttitle = {National Memory},
  title      = {National Memory and Where to Find It},
}
@collection{wagoner2017,
  options    = {skipbib=1},
  editor     = {Wagoner, Brady},
  booktitle  = {Handbook of Culture and Memory},
  date       = {2017},
  shorttitle = {Handbook of Culture and Memory},
  title      = {Handbook of Culture and Memory},
  xdata      = {pub_oxford},
}
\end{filecontents}
\usepackage[style=sbl]{biblatex}
\addbibresource{\jobname.bib}
\begin{document}
\null\vfill
Filler text \autocite{redfield1956}.
Filler text.\footnote{As coined by \citeauthor{redfield1956} in
\cite*[41-42]{redfield1956}. On the topic of national memory, see
\cites[283-303]{carretero-alphen_wagoner2017}[259-281]{wertsch_wagoner2017}.}
\printbibliography
\end{document}
```

![temp468](https://user-images.githubusercontent.com/6268044/60398024-de61e980-9b92-11e9-8333-619bf594d476.png)

**Comment by dcpurton** [comment]
For a temporary fix until `biblatex-sbl` 0.12 is released, you can add this to your preamble:

```TeX
\DefineBibliographyExtras{american}{%
  \DeclarePunctuationPairs{comma}{*!?}%
}
```

**Comment by jackweinbender** [comment]
You're a mensch.
